content/docs/2.15/operate/_index.md

```diff
@@ -8,7 +8,7 @@ We provide guidance and requirements around various areas to operate KEDA:
 
 - Admission Webhooks ([link](./admission-webhooks))
 - Cluster ([link](./cluster))
-- Kubernetes Events ([link](./events))
```

Comment:
Same here, i'm not sure where this should go, so I've removed the link.

Reply:
This one I'm sure of. I moved the events to the reference directory.